tests/test_integrations.py

```diff
@@ -242,7 +251,7 @@ def test_update_license_key(success_mock):
 
 
 @patch("newrelic_lambda_cli.integrations.success")
-def test_install_license_key_already_installed(success_mock):
+def test__install_license_key__already_installed(success_mock):
```

Comment:
Is the double underscore between `key` and `already` intended?

Reply:
Yes. It's just for readability /searchability. We have multiple functions with similar enough names.

`test_<function being tested>__<what is being tested>`